src/class-fields/propname-error/cls-decl-computed-name.template

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2017 Valerie Young. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-class-definitions-static-semantics-early-errors
+path: language/statements/class/fields-computed-name-
+name: computed
+info: |
+  Static Semantics: PropName
+  ...
+  ComputedPropertyName : [ AssignmentExpression ]
+    Return empty.
+---*/
```

Comment:
this template is missing the negative flags.

@rwaldron this should answer [your comment](https://github.com/tc39/test262/pull/1309#discussion_r147448551)